<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.40">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Our research – BrykLab</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<script src="../site_libs/quarto-listing/list.min.js"></script>
<script src="../site_libs/quarto-listing/quarto-listing.js"></script>
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-549806ee2085284f45b00abea8c6df48.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap-d1b12f2568ecbe55642fee6aa00bd082.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script>

  window.document.addEventListener("DOMContentLoaded", function (_event) {
    const listingTargetEl = window.document.querySelector('#listing-listing .list');
    if (!listingTargetEl) {
      // No listing discovered, do not attach.
      return; 
    }

    const options = {
      valueNames: [{ data: ['index'] },{ data: ['categories'] },{ data: ['listing-date-sort'] },{ data: ['listing-file-modified-sort'] }],
      
      searchColumns: [],
    };

    window['quarto-listings'] = window['quarto-listings'] || {};
    window['quarto-listings']['listing-listing'] = new List('listing-listing', options);

    if (window['quarto-listing-loaded']) {
      window['quarto-listing-loaded']();
    }
  });

  window.addEventListener('hashchange',() => {
    if (window['quarto-listing-loaded']) {
      window['quarto-listing-loaded']();
    }
  })
  </script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

<link rel="stylesheet" href="../styles.css">
<link rel="alternate" type="application/rss+xml" title="BrykLab" href="index.xml" data-external="1"></head>

<body class="floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="../index.html">
    <span class="navbar-title">BrykLab</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="../about/index.html"> 
<span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../teaching/index.html"> 
<span class="menu-text">Teaching</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="../research/index.html" aria-current="page"> 
<span class="menu-text">Research</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../stlt/index.html"> 
<span class="menu-text">STLT</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../unigems/index.html"> 
<span class="menu-text">UNIGEMS</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../resources/index.html"> 
<span class="menu-text">Resources</span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://scicomm.xyz/@jarek"> <i class="bi bi-mastodon" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/jarekbryk"> <i class="bi bi-github" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://www.linkedin.com/in/"> <i class="bi bi-linkedin" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="mailto:j.bryk@hud.ac.uk"> <i class="bi bi-envelope" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="../index.xml"> <i class="bi bi-rss" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#ongoing-projects" id="toc-ongoing-projects" class="nav-link active" data-scroll-target="#ongoing-projects">Ongoing projects</a>
  <ul class="collapse">
  <li><a href="#phylogeography-of-apodemus-mice" id="toc-phylogeography-of-apodemus-mice" class="nav-link" data-scroll-target="#phylogeography-of-apodemus-mice">Phylogeography of <em>Apodemus</em> mice</a></li>
  <li><a href="#genetics-and-energetics-in-apodemus-mice" id="toc-genetics-and-energetics-in-apodemus-mice" class="nav-link" data-scroll-target="#genetics-and-energetics-in-apodemus-mice">Genetics and energetics in <em>Apodemus</em> mice</a></li>
  </ul></li>
  <li><a href="#ongoing-small-projects" id="toc-ongoing-small-projects" class="nav-link" data-scroll-target="#ongoing-small-projects">Ongoing small projects</a>
  <ul class="collapse">
  <li><a href="#analysis-of-non-target-sequences-in-a-large-scale-rad-seq-experiment" id="toc-analysis-of-non-target-sequences-in-a-large-scale-rad-seq-experiment" class="nav-link" data-scroll-target="#analysis-of-non-target-sequences-in-a-large-scale-rad-seq-experiment">Analysis of non-target sequences in a large-scale RAD-seq experiment</a></li>
  </ul></li>
  <li><a href="#r-based-resources-to-support-teaching" id="toc-r-based-resources-to-support-teaching" class="nav-link" data-scroll-target="#r-based-resources-to-support-teaching">R-based resources to support teaching</a></li>
  <li><a href="#completed-projects" id="toc-completed-projects" class="nav-link" data-scroll-target="#completed-projects">Completed projects</a>
  <ul class="collapse">
  <li><a href="#seasonal-coat-colour-change-in-mustela-nivalis-weasels" id="toc-seasonal-coat-colour-change-in-mustela-nivalis-weasels" class="nav-link" data-scroll-target="#seasonal-coat-colour-change-in-mustela-nivalis-weasels">Seasonal coat colour change in <em>Mustela nivalis</em> weasels</a></li>
  <li><a href="#analysis-of-sequencing-reads-with-chimeric-adapters-in-a-large-scale-multiplexing-rna-seq-experiment" id="toc-analysis-of-sequencing-reads-with-chimeric-adapters-in-a-large-scale-multiplexing-rna-seq-experiment" class="nav-link" data-scroll-target="#analysis-of-sequencing-reads-with-chimeric-adapters-in-a-large-scale-multiplexing-rna-seq-experiment">Analysis of sequencing reads with chimeric adapters in a large-scale-multiplexing RNA-seq experiment</a></li>
  <li><a href="#experimental-teaching-resources-in-synthetic-biology" id="toc-experimental-teaching-resources-in-synthetic-biology" class="nav-link" data-scroll-target="#experimental-teaching-resources-in-synthetic-biology">Experimental teaching resources in&nbsp;synthetic biology</a></li>
  </ul></li>
  <li><a href="#rabbits-friends-and-relations" id="toc-rabbits-friends-and-relations" class="nav-link" data-scroll-target="#rabbits-friends-and-relations">Rabbit’s Friends and Relations</a></li>
  <li><a href="#funders" id="toc-funders" class="nav-link" data-scroll-target="#funders">Funders</a></li>
  </ul>
</nav>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Our research</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="ongoing-projects" class="level2">
<h2 class="anchored" data-anchor-id="ongoing-projects">Ongoing projects</h2>
<section id="phylogeography-of-apodemus-mice" class="level3">
<h3 class="anchored" data-anchor-id="phylogeography-of-apodemus-mice">Phylogeography of <em>Apodemus</em> mice</h3>
<p>Rodents from the genus <em>Apodemus</em> are the most common mammals of the Paleoarctic region and the main interest of our group. They occupy environments as different as Spain and Siberia, contribute to spread of human diseases like <em>Lyme borreliosis</em> and tick-borne encephalitis and are a rich target for evolutionary studies. However, they are very underdeveloped in terms of their genomic and genetic resources. Our long-term goal is to help establish <em>Apodemus</em> as a full–blown model organism and utilise its generalist ecology for studies of evolution in the wild.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../../images/apodemus_flavicolis.jpg" title="Apodemus flavicolis; photograph © Karol Zub" class="img-fluid figure-img"></p>
<figcaption><em>Apodemus flavicolis</em>; photograph © Karol Zub</figcaption>
</figure>
</div>
<p>The major project we are currently undertaking is on improving the European phylogeography of the two species, <em>Apodemus flavicolis</em> (yellow-necked mouse, also confusingly called wood mouse in Polish) and <em>Apodemus sylvaticus</em> (wood mouse). The goal is to collect and describe large(ish)-scale (n &gt; 350 samples) data on genetic diversity and connectivity of the species and help answer some of the mysteries of recolonisation of Europe by small mammals following the last glaciation. The project is helmed by Maria Luisa Martin Cerezo and utilises restriction-site-associated DNA sequencing (RAD-seq) to obtain whole-genome, high-density genotypes even without a reference genome. It has been possible thanks to a number of colleagues across Europe, from Scotland and Portugal through Serbia to Lithuania, who provided samples from across the range of both species.</p>
<p>The first publication from this project, a small-scale phylogeographic analysis of Polish populations of <em>Apodemus</em>, has been published in 2020: <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-6603-3">Population structure of <em>Apodemus flavicollis</em> and comparison to <em>Apodemus sylvaticus</em> in northern Poland based on RAD-seq</a>.</p>
</section>
<section id="genetics-and-energetics-in-apodemus-mice" class="level3">
<h3 class="anchored" data-anchor-id="genetics-and-energetics-in-apodemus-mice">Genetics and energetics in <em>Apodemus</em> mice</h3>
<p>The second project, run by Rohan Raval in collaboration with Drs Karol Zub and Jan Boratyński at the <a href="http://www.zbs.bialowieza.pl/ibs_pan/">Mammal Research Institute of the Polish Academy of Sciences</a> (MRI PAS) in <a href="http://whc.unesco.org/en/list/33/">Białowieża</a>, Poland, will characterise the patterns of metabolic rates in a single population of <em>A. flavicolis</em> continually sampled for multiple seasons. The project involves genotyping and pedigree analysis of a large, wild population of mice and lab-based measures of basal metabolic rates (BMR) and torpor. Our goal is to measure heritability of BMR in the wild throughout multiple seasons as well as in-depth characterisation of population structure and dynamics.</p>
</section>
</section>
<section id="ongoing-small-projects" class="level2">
<h2 class="anchored" data-anchor-id="ongoing-small-projects">Ongoing small projects</h2>
<p>A set of projects that get pushed forward only when me and willing students have enough time to devote to them.</p>
<section id="analysis-of-non-target-sequences-in-a-large-scale-rad-seq-experiment" class="level3">
<h3 class="anchored" data-anchor-id="analysis-of-non-target-sequences-in-a-large-scale-rad-seq-experiment">Analysis of non-target sequences in a large-scale RAD-seq experiment</h3>
<p>After amassing data from almost 1500 samples of <em>Apodemus</em> mice, we realised that a non-trivial proportion of reads we collect come not from <em>Apodemus</em> but other organisms “contaminating” the samples. We were keen to see whether sequences of some of the parasites and vectors that <em>Apodemus</em> carry, such as ticks, <em>Borrelia</em> or hantaviruses could be retrieved from these carryover sequences. After Marisa checked that, indeed, <em>Borrelia</em> and <em>Ixodes</em> could be sequenced in our <em>Apodemus</em> RAD-seq protocol, Haeyam took over and during her placement she established a proof-of-principle, Kraken-based screen for microbial and insect sequences in our samples. This original idea was subsequently worked on by Ray (Wayne-hon Lau, my once placement student) to turn it into a proper pipeline to use in our future experiments, RAD-seq or otherwise.</p>
</section>
</section>
<section id="r-based-resources-to-support-teaching" class="level2">
<h2 class="anchored" data-anchor-id="r-based-resources-to-support-teaching">R-based resources to support teaching</h2>
<section id="maudr" class="level4">
<h4 class="anchored" data-anchor-id="maudr">maudR</h4>
<p>Some time during the pandemic lockdown phase, <a href="https://pure.hud.ac.uk/en/persons/richard-bingham">Dr Richard Bingham</a> had an idea about generating a set of datasets that simulate a typical enzyme activity practical in undergraduate biology laboratory. He wrote an initial code for alcohol dehydrogenase and a series of its inhibitors, which I then expanded and scripted to also automatically generate a sesies of A4 PDFs with individual solutions to students’ tasks (absorbance vs.&nbsp;time, Michaelis-Menten and Lineweaver-Burke plots).</p>
<p>After contributions from Dr Shamus Burns, Wayne-hon Lau and Lewis Ward, this code is now being turned into an R package, soon to be released on Github. Lewis and me are both using maudR to learn how to write an R package; Lewis presented this idea as a poster during the <a href="https://mlindner0.github.io/PopulationGenetics58Website/program/">PopGroup UK 58 meeting in Sheffield in 2025</a>.</p>
<p>I presented this idea during the <a href="https://northernbug.github.io/northernbug9">nBUG 9 meeting in Liverpool</a> and during the useR! conference in Salzburg in 2024. <a href="https://userconf2024.sched.com/event/1c8vr/datasets-and-assignments-for-undergraduate-teaching-enzyme-kinetics-as-an-example-jarek-bryk-university-of-huddersfield">Here are my (static) slides from the talk</a> and a <a href="https://github.com/jarekbryk/maudr">Github repo where you can download the R project files</a> to use it (not a package yet though!).</p>
</section>
<section id="huddr" class="level4">
<h4 class="anchored" data-anchor-id="huddr">huddR</h4>
<p>In 2025, a group of keen 2nd year undergraduate students (Ana, Chantay, Fern, Layane and Sofia) got together in the revived Coding Club to complete a long-standing idea of mine: to produce an online textbook to support teaching of R programming to our 2nd year biology students. Our goal is to have something like the wonderful <a href="https://psyteachr.github.io">PsyTeachR resources</a>, particularly their <a href="https://psyteachr.github.io/data-skills-v3/">Data Skills book</a>, but tailored to what and how we teach students. So far (I am writing this in February, one month after we started), the <a href="huddr.netlify.app/">very rough draft of the book is out there</a> and it will only get better from there.</p>
<p>We (myself included) will use this project also to learn version control through Git and Github and how to collaborate on a coding project.</p>
</section>
<section id="easy-pca-and-admixture-plots-from-1000-genomes-project-in-r" class="level4">
<h4 class="anchored" data-anchor-id="easy-pca-and-admixture-plots-from-1000-genomes-project-in-r">Easy PCA and ADMIXTURE plots from 1000 Genomes Project in R</h4>
<p>TBA</p>
</section>
<section id="semi-automatic-phylogenetic-trees-from-bulk-ncbi-sequences" class="level4">
<h4 class="anchored" data-anchor-id="semi-automatic-phylogenetic-trees-from-bulk-ncbi-sequences">Semi-automatic phylogenetic trees from bulk NCBI sequences</h4>
<p>TBA</p>
</section>
</section>
<section id="completed-projects" class="level2">
<h2 class="anchored" data-anchor-id="completed-projects">Completed projects</h2>
<section id="seasonal-coat-colour-change-in-mustela-nivalis-weasels" class="level3">
<h3 class="anchored" data-anchor-id="seasonal-coat-colour-change-in-mustela-nivalis-weasels">Seasonal coat colour change in <em>Mustela nivalis</em> weasels</h3>
<p>There are two subspecies of least weasels in north–eastern Poland. <em>Mustela nivalis nivalis</em> and <em>Mustela nivalis vulgaris</em>. They live in relative sympatry (<em>M. n.&nbsp;vulgaris</em> is a recent invader from the south) and are similarly-looking during most of the year. However, when the days start to get shorter, the coat of the <em>M. n.&nbsp;nivalis</em> changes to completely white, providing it with camouflage and increased survival on snow cover, whereas the coat of the <em>M. n.&nbsp;vulgaris</em> stays a similar shade of brown during the entire year.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../../images/mustela_nivalis.jpg" title="Least weasels in the summer and winter; photograph © Karol Zub" class="img-fluid figure-img"></p>
<figcaption>Least weasels in the summer and winter; photograph © Karol Zub</figcaption>
</figure>
</div>
<p>We are collaborating with Dr Karol Zub at MRI PAS and <a href="https://cibio.up.pt/people/details/jmferrei">Dr José Melo-Ferreira</a> at the <a href="https://cibio.up.pt/about">Research Centre in Biodiversity and Genetic Resources</a> at the <a href="https://sigarra.up.pt/up/en/web_base.gera_pagina?p_pagina=home">University of Porto</a> on elucidating the molecular and evolutionary forced underlying the seasonal colour change.</p>
<p>We started from Sanger–sequencing a host of putative candidate genes affecting coat colour in order to exclude a possibility of stable coding-sequence polymorphism affecting coat colour change, however, we were unable to proceed much further due to lack of fresh samples of <em>Mustela</em>. Fortunately, our colleagues in Portugal were <em>much</em> more successful with their high-throughput sequencing of museum samples.</p>
<p>The paper has now been published in MBE (open access!): <a href="https://academic.oup.com/mbe/article/38/10/4388/6304872">Museomics dissects the genetic basis for adaptive seasonal colouration in the least weasel</a>.</p>
</section>
<section id="analysis-of-sequencing-reads-with-chimeric-adapters-in-a-large-scale-multiplexing-rna-seq-experiment" class="level3">
<h3 class="anchored" data-anchor-id="analysis-of-sequencing-reads-with-chimeric-adapters-in-a-large-scale-multiplexing-rna-seq-experiment">Analysis of sequencing reads with chimeric adapters in a large-scale-multiplexing RNA-seq experiment</h3>
<p>In the course of some of the research described above, we designed and tested sequencing adapters and library preparation methods that enabled us to characterise the types of polymerase mishaps that happen whenever multiple different templates are being amplified together. Marisa and Rohan have been working on analysing various types of chimeric reads that are generated during our large-scale experiments with a view to improve existing protocols to minimise the influence of such reads on genotyping.</p>
<p>The paper has now been published in Molecular Ecology Resources (open access!): <a href="https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.13661">Identification and quantification of chimeric sequencing reads in a highly multiplexed RAD-seq protocol</a>.</p>
</section>
<section id="experimental-teaching-resources-in-synthetic-biology" class="level3">
<h3 class="anchored" data-anchor-id="experimental-teaching-resources-in-synthetic-biology">Experimental teaching resources in&nbsp;synthetic biology</h3>
<p><a name="synbio"></a></p>
<p><em>This project is completed, but every now and then we attempt to design and test new parts for UNIGEMS. If we succeed, they will be deposited in Addgene.</em></p>
<p>Synthetic biology has enjoyed an explosive growth and a dramatic rise in prominence in the last decade, but the branch of synthetic biology that focuses on standardisation of DNA parts and thorough characterisation of genetic circuits still appears to be somewhat less than what it should be (with the fantastic exception of the <a href="http://igem.org/Main_Page">International Genetically Engineered Machines</a> competition). There is still a lack of well–characterised and tested sets of parts that allow for introducing synthetic biology concepts and techniques to secondary schools.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../../images/plasmid_fragment.gif" title="Fragment of a plasmid diagram showing GFP with a T5/lac promoter © Jarek Bryk" class="img-fluid figure-img"></p>
<figcaption>Fragment of a plasmid diagram showing GFP with a T5/lac promoter © Jarek Bryk</figcaption>
</figure>
</div>
<p><a href="http://www.ncbe.reading.ac.uk">National Centre for Biotechnology Education</a> at the <a href="http://www.reading.ac.uk">University of Reading</a> have been working on a Wellcome Trust–funded project to develop experimental resources in synthetic biology for the A-level education, following our previous work on the <a href="../unigems/">UNIGEMS project</a>. This project was completed by Alexandra Siddall and the set of 10 plasmids with various DNA parts is available at <a href="addgene.org/Jaroslaw_Bryk/">http://www.addgene.org/Jaroslaw_Bryk/</a>.</p>
<p>The paper has now been published: <a href="https://www.microbiologyresearch.org/content/journal/acmi/10.1099/acmi.0.000596.v3">Unigems: plasmids and parts to facilitate teaching on assembly, gene expression control and logic in <em>E. coli</em>.</a>; <a href="https://figshare.com/projects/Unigems_paper/114069">extra materials are available on Figshare</a> and the <a href="https://www.addgene.org/Jaroslaw_Bryk/">plasmids are available at Addgene</a>. <a href="https://www.addgene.org/Jaroslaw_Bryk/"><img src="https://www.addgene.org/headers/images/plasmid_request_button_no_pi.png" alt="Link to BrykLab's plasmids at Addgene"></a></p>
</section>
</section>
<section id="rabbits-friends-and-relations" class="level2">
<h2 class="anchored" data-anchor-id="rabbits-friends-and-relations">Rabbit’s Friends and Relations</h2>
<p>Our research would not be possible without these fantastic collaborators:</p>
<ul>
<li><p>Dr Frank Yinnguan Chan and Marek Kucka from the <a href="http://www.fml.tuebingen.mpg.de">Friedrich Miescher Laboratory of the Max Planck Society</a> in Tübingen, Germany (sequencing, library preparation). Frank is currently at the <a href="https://www.rug.nl/staff/frank.chan/?lang=en">University of Groningen</a>.</p></li>
<li><p><a href="http://www.ibs.bialowieza.pl/artykul/405.html">Dr Karol Zub</a> and Dr Jan Boratyński from the <a href="http://www.zbs.bialowieza.pl/ibs_pan/">Mammal Research Institute of the Polish Academy of Sciences</a> (MRI PAS) in <a href="http://whc.unesco.org/en/list/33/">Białowieża</a>, Poland (<em>Apodemus</em> and <em>Mustela</em> samples, energetics, ecology).</p></li>
<li><p><a href="https://cibio.up.pt/people/details/jmferrei">Dr Jose Melo-Ferreira</a> from the <a href="https://cibio.up.pt/about">Research Centre in Biodiversity and Genetic Resources</a> at the <a href="https://sigarra.up.pt/up/en/web_base.gera_pagina?p_pagina=home">University of Porto</a>, Portugal (<em>Mustela</em> coat colouration).</p></li>
<li><p><a href="https://www.liverpool.ac.uk/integrative-biology/staff/stephen-paterson/">Dr Steve Patterson</a> from the <a href="https://www.liverpool.ac.uk">University of Liverpool</a>, UK (<em>Apodemus</em> 10x Genomics sequencing, genome annotation)</p></li>
<li><p><a href="https://klochlab.eu/index.php/author/owca/">Dr Agnieszka Kloch</a> from the <a href="https://en.uw.edu.pl">University of Warsaw</a>, Poland (RAD-seq, <em>Microtus</em> and their parasites).</p></li>
<li><p><a href="https://staffprofiles.bournemouth.ac.uk/display/ehardouin">Dr Emilie Hardouin</a> from the <a href="https://www.bournemouth.ac.uk">University of Bournemouth</a>, UK (RAD-seq, foxes and squirrels).</p></li>
<li><p>Dr Fiona Lane and Dr John Schollar from the <a href="http://www.ncbe.reading.ac.uk">National Centre for Biotechnology Education</a> at the <a href="http://www.reading.ac.uk">University of Reading</a>, UK (synthetic biology, education).</p></li>
<li><p><a href="https://pure.hud.ac.uk/en/persons/vicki-trowler">Dr Vicki Trowler</a> and <a href="https://pure.hud.ac.uk/en/persons/john-stephenson">Dr John Stephenson</a> from the <a href="https://www.hud.ac.uk">University of Huddersfield</a> (student attainment project).</p></li>
<li><p>Many others, who provided samples and/or expertise on <em>Apodemus</em> phylogeography, sample preparation and/or data analysis, especially: Dr Jochan Michaux, <a href="https://mfrc-atu.ie/meet-the-team/allan-mcdevitt/">Dr Allan McDevitt</a>, <a href="https://www.nms.ac.uk/collections-research/collections-departments/natural-sciences/vertebrate-biology/dr-jerry-herman/">Dr Jeremy Herman</a>, Dr Vladimir Jovanovic.</p></li>
</ul>
</section>
<section id="funders" class="level2">
<h2 class="anchored" data-anchor-id="funders">Funders</h2>
<p>Our work has been supported by the <a href="https://www.hud.ac.uk">University of Huddersfield</a>, <a href="http://www.bbsrc.ac.uk/funding/filter/stars/">BBSRC STARS</a> programme, <a href="https://www.microsoft.com/en-us/research/academic-program/microsoft-azure-for-research/">Microsoft Azure for Research</a>, <a href="https://cloud.google.com">Google Cloud Platform</a> and the <a href="http://nbaf.nerc.ac.uk">NERC Biomolecular Analysis Facility</a>.</p>



</section>

<div class="quarto-listing quarto-listing-container-default" id="listing-listing">
<div class="list quarto-listing-default">

</div>
<div class="listing-no-matching d-none">
No matching items
</div>
</div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp("https:\/\/bryklab\.net");
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>